README.md

```diff
@@ -16,51 +16,99 @@ If you're having trouble getting set up, reach out to us at [our mailing list](h
 Setup
 -----
 0. Learn a bit about Git and GitHub:
+	* Install git on your PATH
 	* [GitHub help](https://help.github.com/)
 	* [Try Git](https://try.github.io/)
-1. Install [App Engine](http://developers.google.com/appengine/)
-	* Specifically use the [Python SDK](http://developers.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python)
-	* Run installer and allow it make symbolic links (you will be asked to enter your root password)
+1. Install [Python 2.7.X](https://www.python.org/downloads/)
+1. Install [App Engine](https://cloud.google.com/appengine/docs)
+	* Specifically use the [Python SDK](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python)
+	* Run the installer and allow it make symbolic links (it might ask you to enter your root password)
 2. Get the latest version of The Blue Alliance
-	* First make sure you have forked TBA by clicking on "Fork" in the top right of the page
-	* Run `git clone git://github.com/username/the-blue-alliance.git` where username is your GitHub username
+	* Fork TBA by clicking on "Fork" in the top right of [its GitHub page](https://github.com/the-blue-alliance/the-blue-alliance)
+	* Run `git clone git://github.com/USERNAME/the-blue-alliance.git` where _USERNAME_ is your GitHub username, or use GitHub's Windows or OS X app to clone it to your computer
 	* For detailed instructions see [the GitHub guide on contributing](https://guides.github.com/activities/contributing-to-open-source/index.html#contributing)
-3. Import the project into Google App Engine Launcher
-	* By default TBA uses port **8088**, make sure your local setup is consistent with this
-4. Run the app in App Engine
-5. Get some test data: In your terminal console, from the `the-blue-alliance` directory, run the following command
-	* `paver setup` (Make sure that you already have [Paver installed](#paver-commands))
-6. You should now have a basic development installation!
+3. Install [numpy](http://www.numpy.org/)
```

Comment:
Nice! And then the package manager will also make updates easy.

OK, so I need to move off Windows. It'll have to be OS X since Lightroom won't run on Linux.